tokio-trace-futures/src/test_support/field.rs

```diff
@@ -199,12 +193,11 @@ impl<'a> From<&'a Value> for MockValue {
             }
         }
 
-        let fake_field =
-            callsite!(name: "fake", kind: tokio_trace::metadata::Kind::EVENT, fields: fake_field)
-                .metadata()
-                .fields()
-                .field("fake_field")
-                .unwrap();
+        let fake_field = callsite!(name: "fake", kind: Kind::EVENT, fields: fake_field)
```

Comment:
this is the part that actually got fixed, for some reason it wouldn't compile until I  moved the include for  Kind to t he top of the file.